### PR TITLE
docs: update hugo to latest version 0.69.2

### PR DESCRIPTION
**Pull request by tumarsal**

### CODE REVIEW

File: docs-src/config.toml [hunk 1]
```diff
@@ -39,3 +39,6 @@ name = "<i class='fa fa-bookmark'></i> <label>Contact us</label>"
 identifier = "contactus"
 url = "https://www.cobaltspeech.com/contact"
 weight = 20
+
+[markup.goldmark.renderer]
+unsafe= true
```
Comment (mplooster) [inline docs-src/config.toml:44]: What does this flag do?

File: docs-src/content/using-diatheke-sdk/session/audio-input.md [hunk 2]
```diff
@@ -13,19 +13,18 @@ create the input stream. For a given session, only a single audio input
 stream should be open at a time, but it perfectly acceptable to open
 and close multiple audio streams over the course of a single session.
 
-{{%tabs %}}
+{{< tabs >}}
 
-{{% tab "Go" %}}
+{{< tab "Go" "go" >}}
 ``` go
```
Comment (mplooster) [inline docs-src/content/using-diatheke-sdk/session/audio-input.md:19]: You forgot to remove the ` ``` go` bit here.

File: docs-src/content/using-diatheke-sdk/session/audio-input.md [hunk 3]
```diff
@@ -13,19 +13,18 @@ create the input stream. For a given session, only a single audio input
...
 
 // OR create the stream using the Session object
 stream, err := session.StreamAudioInput(context.Background())
-```
-{{% /tab %}}
+{{< /tab >}}
 
-{{% tab "C++" %}}
+{{< tab "C++" "c++" >}}
 ``` c++
```
Comment (mplooster) [inline docs-src/content/using-diatheke-sdk/session/audio-input.md:28]: Remove the ` ``` c++` part.

File: docs-src/content/using-diatheke-sdk/session/audio-input.md [hunk 4]
```diff
@@ -34,20 +33,18 @@ stream = client.streamAudioInput(sessionID);
 
 // OR create the stream using the Session object
 stream = session.streamAudioInput();
-```
-{{% /tab %}}
+{{< /tab >}}
 
-{{% tab "Python" %}}
+{{< tab "Python" "python">}}
 ``` python
```
Comment (mplooster) [inline docs-src/content/using-diatheke-sdk/session/audio-input.md:39]: Remove the ` ``` python` part.

File: docs-src/content/using-diatheke-sdk/session/audio-input.md [hunk 5]
```diff
@@ -34,20 +33,18 @@ stream = client.streamAudioInput(sessionID);
...
 
 # OR create the stream using the Session object
 stream = session.stream_audio_input()
-```
-{{% /tab %}}
+{{< /tab >}}
 
-{{% tab "Swift/iOS" %}}
+{{< tab "Swift/iOS" "swift" >}}
 ``` swift
```
Comment (mplooster) [inline docs-src/content/using-diatheke-sdk/session/audio-input.md:48]: Remove the ` ```swift` part.

File: docs-src/config.toml (hunk 1 above)
Comment (tumarsal) [reply docs-src/config.toml:44]: I don't know. (I remove that)

File: docs-src/layouts/shortcodes/tab.html [hunk 6]
```diff
@@ -1,3 +1,3 @@
 <div class="tab-pane" title="{{ .Get 0 }}">
-  {{ .Inner }}
+  {{ highlight .Inner (.Get 1)  "" }}
```
Comment (mplooster) [inline docs-src/layouts/shortcodes/tab.html:2]: Why this change?

Comment (mplooster) [reply docs-src/layouts/shortcodes/tab.html:2]: Ah, nevermind. I think I get it.
